### PR TITLE
Adjust font-size of Jobs Dashboard components

### PR DESCRIPTION
**Pull request by bernardodiasc**
**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3497

## Description

Adjust font-size of Jobs Dashboard components.

Attention during review: the majority of the text elements doesn't have a representation for the mobile version in Figma. I made adjustments from my gut feeling trying to preserve harmony and somehow a consistency with the other text elements that exists in Figma. 

## Checklist

- [x] set yourself as an assignee
- [x] set appropriate labels for a PR (`In Review` or `In Progress` depending on its status)
- [x] move respective JIRA issue to the `PEER REVIEW` column
- [x] make sure your code lints (`npm run lint`)
- [x] Flow typechecks passed (`npm run flow`)
- [x] once finished with all the changes, you run `npm run build` command and committed changed files to the branch
- [x] **manually tested the app** by running it in several different browsers (Firefox, Chrome, Opera, Safari, MS IE/Edge, etc.) and checked nothing is broken and operates as expected!

## Steps to Test or Reproduce

Check mobile viewport for the components:

- http://localhost:9001/?path=/story/ui-components-closedjobapplications--basic-usage
- http://localhost:9001/?path=/story/ui-components-headerbar-use-cases--with-links
- http://localhost:9001/?path=/story/ui-components-jobapplicationcard--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobcard--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobdetails--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobskills--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobspagebreadcrumbs--basic-usage
- http://localhost:9001/?path=/story/ui-components-welcomehero--basic-usage
- http://localhost:9001/?path=/story/ui-components-emptystate--basic-usage
- http://localhost:9001/?path=/story/ui-components-applicantscreennotification--basic-usage
- http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout--basic-usage
- http://localhost:9001/?path=/story/core-components-confirmationbox--basic-usage
- http://localhost:9001/?path=/story/core-components-errorbox--basic-usage

## Impacted Areas in Application

- `ClosedJobApplications`
- `HeaderBar`
- `JobApplicationCard`
- `JobCard`
- `JobDetails`
- `JobSkills`
- `JobsPageBreadcrumbs`
- `WelcomeHero`
- `EmptyState`
- `ApplicantScreenNotification`
- `JobsPageLayout`
- `ConfirmationBox`
- `ErrorBox`

### CODE REVIEW

**Comment by bernardodiasc** [comment]
I think ErrorBox didn't got any change. Just need to check how it looks because is one of the UI components used in XP-Reg.

File: src/components/ui/EmptyState.js [hunk 1]
```diff
@@ -33,6 +33,11 @@ const cx = {
       & a:hover {
         color: ${theme.typoAnchorHover}
       }
+
+      @media screen and (min-width: ${breakpoints.sm}) {
+        & {
```
Comment (nicksp) [inline src/components/ui/EmptyState.js]: Is this really needed? 🤔  Let's clean up otherwise.

**Comment by nicksp** [comment]
> I think ErrorBox didn't got any change.

Could you please remove it from the `Impacted Areas in App` then to be up-to-date?

**Comment by bernardodiasc** [comment]
Those components got more updates based on the feedback at https://github.com/x-team/xp-registration/pull/129#pullrequestreview-345194396

Screen Shot 2020-01-20 at 07 33 24

Screen Shot 2020-01-20 at 07 19 11

Screen Shot 2020-01-20 at 07 19 16

Screen Shot 2020-01-20 at 07 20 25

Screen Shot 2020-01-20 at 07 25 46

Screen Shot 2020-01-20 at 07 25 57

Screen Shot 2020-01-20 at 07 28 08

For the `ErrorBox` I made an adjustment for the font-size to be a bit smaller on mobile.

**Comment by nicksp** [comment]
FYI: I removed `ErrorBox` from the `Impacted Areas in the App` as I mentioned above.

**Comment by bernardodiasc** [comment]
> FYI: I removed ErrorBox from the Impacted Areas in the App as I mentioned above.

But I included changes in that component, I mentioned that in my previous comment.

**Comment by nicksp** [comment]
> But I included changes in that component, I mentioned that in my previous comment.

Ah, nvm 👍 Might have missed that.

**Pull request merged**